006_plug_and_play_prior/toto.ipynb

```diff
@@ -0,0 +1,705 @@
+      "text/plain": [
+       "<Figure size 1000x1000 with 2 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Observation\n",
+    "# need to specify a prior --> Gaussian as not informative as possible\n",
+    "X_generation = Gaussian(0, gamma2, geometry = geometry)\n",
+    "Y_generation = Gaussian(A_op @ X_generation, sigma2, geometry = geometry)\n",
+    "y_data = Y_generation(X_generation = im).sample()\n",
+    "\n",
+    "fig = plt.figure(figsize = (10, 10))\n",
+    "ax1 = fig.add_subplot(1, 2, 1)\n",
+    "ax1.imshow(im, cmap = \"gray\")\n",
+    "ax1.axis(\"off\")\n",
+    "ax1.set_title(\"Original image\")\n",
+    "ax2 = fig.add_subplot(1, 2, 2)\n",
+    "ax2.imshow(np.reshape(y_data, (N, N)), cmap = \"gray\")\n",
+    "ax2.set_title(\"Observation\")\n",
+    "ax2.axis(\"off\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Loading the Plug-and-Play regularizer\n",
+    "\n",
+    "We first load the pre-trained denoiser which implicitly defines a regularizer. It has been trained at noise level $s=2.25$."
```

Comment:
I think it would be useful in this section to do a basic demonstration of the denoiser by applying it directly (without any CUQIpy) to a noisy image and displaying the resulting denoised image along with the noisy image. This will demonstrate what the denoiser does, and also that it does not do any deblurring, only denoising.